platform/view/sdk/sdk.go

```diff
@@ -452,10 +454,17 @@ func (p *SDK) initWebOperationEndpointsAndMetrics() error {
 		}
 	}
 
+	prometheusTls := true
+	if configProvider.IsSet("fsc.metrics.prometheus.tls") {
+		prometheusTls = configProvider.GetBool("fsc.metrics.prometheus.tls")
+	}
+	logger.Infof("Starting prometheus with TLS: %v", prometheusTls)
```

Comment:
but what if the provider is not prometheus?

Reply:
then the prometheus section in the config will not be defined, and the TLS option will default to false, as is the case right now that the variable doesn't exist.

Reply:
but it will still log `"Starting prometheus with TLS`.

Reply:
oh indeed let me fix it